src/scripts/modules/footer/footer-template.html

```diff
@@ -22,9 +22,7 @@
   <section class="connect">
     <h1 class="small-header" data-l10n-id="all-footer-connect">Connect</h1>
     <ul>
-      <li><a href="https://play.google.com/store/apps/details?id=org.cnx.android" data-l10n-id="all-footer-android-app">Android App</a></li>
-      <li><a href="//devblog.cnx.org" data-l10n-id="all-footer-dev-blog">Dev Blog</a></li>
-      <li><a href="http://itunes.apple.com/us/institution/connexions/id375198208" data-l10n-id="all-footer-itunes">iTunes U</a></li>
+      <li><a href="https://play.google.com/store/apps/details?id=org.openstaxcollege.android&hl=en" data-l10n-id="all-footer-android-app">Android App</a></li>
```

Comment:
We cannot change the package name. If we did Google would consider it a new app since the package is the identifier. When I first made the app openstax college was the name.

Reply:
Language id has been removed from URL